Aqovia.Utilities.SagaMachine/SagaMachine.cs

```diff
@@ -30,7 +30,7 @@ public SagaMachine(IKeyValueStore keyValueStore, Func<IEnumerable<ISagaMessageId
 
         public async Task Handle<TMessage>(TMessage message) where TMessage : ISagaMessageIdentifier
         {
-            await Task.Run(() => ProcessMessage(message));
+            await Task.Run(() => ProcessMessage(message)).ConfigureAwait(false);
```

Comment:
But this is a library, so it should ensure that callers can't do something stupid. [Recommendation from Stephen Cleary](http://blog.stephencleary.com/2012/07/dont-block-on-async-code.html#preventing-the-deadlock) (MVP who wrote the [MSDN Async programming guide](https://msdn.microsoft.com/en-us/magazine/jj991977.aspx)) is to set ConfigureAwait(false) on "library" functions whenever possible.